### DISCUSSION TITLE
Seata/RM/BranchCommitRequest为什么独立于请求链之外

### DISCUSSION THREAD

**Discussion by Mxgxg** [apache/incubator-seata#4385] [Q&A]
测试的请求是用develop编译的1.5.0快照版本，服务端也是develop打包的，skywalking是8.7版本，用的是AT模式。服务都是注册到nacos中的。
发送请求后追踪如下
![d87447ca9813091340dbf1c13a7186c](https://user-images.githubusercontent.com/79822553/154218395-11d79b07-434d-4e48-8e1d-4ce20aa2325d.png)

项目配置如下：
![c2d733d4d0977c86e91213bc631cdad](https://user-images.githubusercontent.com/79822553/154219431-76c0fdad-6e35-47cb-b7ad-557273ae98c6.png)

![ef05e2bddf864df9efac0ea67c3d51b](https://user-images.githubusercontent.com/79822553/154219441-b4b971e8-3f91-43fc-88b8-0acb3b000837.png)

RM的追踪为什么没在追踪链之内呢。是我哪里配置不对么。

**Comment by slievrly** [comment]
@Mxgxg BranchCommitRequest 是异步执行的，这块tracing链路有问题。
